newIDE/app/src/ResourcesList/ResourceUtils.js

```diff
@@ -80,7 +80,26 @@ export const copyAllToProjectFolder = (
         return resourcePath;
       }
 
-      const resourceBasename = path.basename(resourcePath);
+      let resourceBasename = path.basename(resourcePath),
+        splitAt = resourceBasename.lastIndexOf('.'),
+        fileName = resourceBasename.substring(0, splitAt),
+        fileExtension = resourceBasename.substring(splitAt),
+        fileNumber = 1;
+
+      const AllResourcesNames = project
```

Comment:
This is a normal variable, so this should be named in `camelCase`. `PascalCase` is used for classes only.